web_api/core/views.py

```diff
@@ -211,14 +215,19 @@ def start_trial(request: HttpRequest, team_id: str) -> HttpResponse:
     return HttpResponse(status=204)
 
 
+class UpdateSubscriptionModel(pydantic.BaseModel):
+    seats: int
+    prorationTimestamp: int
+    planPeriod: Literal["month", "year"] = "month"
+
+
 @auth.login_required
 def update_subscription(request: HttpRequest, team_id: str) -> HttpResponse:
+    payload = UpdateSubscriptionModel.parse_obj(request.POST.dict())
```

Comment:
I wonder if there is a better name than `payload`. 🤷‍♂️

Reply:
I think `payload` is fine 

Some other options:

- `body`
- `request_body`
- `req`
- `data`
- `request_data`